node/src/accountant/scanners_tools.rs

```diff
@@ -330,7 +342,7 @@ mod tests {
     }
 
     #[test]
-    fn payable_with_low_debt_is_marked_unqualified() {
+    fn payable_with_debt_smaller_than_permanent_debt_allowed_is_marked_unqualified() {
```

Comment:
Some of these test might be deprecated later when you have the master merged in. I spent a lot of time on testing the thresholds really in deep. So don't hesitate to call me over and I can navigate you through the merge. I'd be tempered if we lost the exacter tests I wrote.